bool RTK_enable
bool enable
bool autonomous
